AnaProd/tasks.py

```diff
@@ -83,14 +83,15 @@ def run(self):
         producer = os.path.join(self.ana_path(), 'AnaProd', 'anaCacheProducer.py')
         input_files = InputFileTask.load_input_files(self.input()[0].path, sample_name)
         ana_caches = []
+        generator_name = self.samples[sample_name]['generator']
```

Comment:
does this work for data?

Reply:
for data it should not arrive to this line but I can fix this in any case just to be sure